server/src/main/java/com/iqas/server/model/KbModel.java

```diff
@@ -1,31 +1,55 @@
 package com.iqas.server.model;
 
+import org.bson.types.ObjectId;
 import org.springframework.data.annotation.Id;
+import org.springframework.data.annotation.Transient;
 import org.springframework.data.mongodb.core.mapping.Document;
+import org.springframework.data.mongodb.core.mapping.Field;
 
+import java.util.ArrayList;
 import java.util.Date;
+import java.util.List;
 
 @Document
 public class KbModel {
 
-    private String kbId;
+    private ObjectId _id;
+    private Integer kbId;
     private String name;
+    @Field
+    private List<Integer> docId;
     private Date creationDate = new Date();
 
     public KbModel() {
     }
 
-    public KbModel(String kbId, String name) {
+    public KbModel(String name) {
         super();
-        this.kbId = kbId;
         this.name = name;
+        this.docId=new ArrayList();
+    }
+
+    public ObjectId get_id() {
+        return _id;
+    }
+
+    public void set_id(ObjectId _id) {
```

Comment:
```suggestion
    public void setId(ObjectId _id) {
```

We should probably use camel case here for consistency :)

Reply:
private ObjectId _id;
is a member of the KbModel class, it is special because '_id' it is storing mongo's default '_id'. If we will change '_id' to 'Id' we would not be able to capture the data. Thus getter and setter seems to be camel case :D

Reply:
https://github.com/divyanshbajpai/iqas/pull/3/commits/4887a40eb48d0ac80f5271c79a60c36c2d160dcb